WordPress/src/main/java/org/wordpress/android/ui/reader/views/ReaderRelatedPostsView.java

```diff
@@ -0,0 +1,216 @@
+package org.wordpress.android.ui.reader.views;
+
+import android.annotation.TargetApi;
+import android.content.Context;
+import android.os.Build;
+import android.support.annotation.NonNull;
+import android.util.AttributeSet;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.ViewTreeObserver;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import org.wordpress.android.R;
+import org.wordpress.android.ui.reader.actions.ReaderActions;
+import org.wordpress.android.ui.reader.actions.ReaderBlogActions;
+import org.wordpress.android.ui.reader.models.ReaderRelatedPost;
+import org.wordpress.android.ui.reader.models.ReaderRelatedPostList;
+import org.wordpress.android.util.DisplayUtils;
+import org.wordpress.android.util.GravatarUtils;
+import org.wordpress.android.util.NetworkUtils;
+import org.wordpress.android.util.PhotonUtils;
+import org.wordpress.android.util.ToastUtils;
+import org.wordpress.android.widgets.WPNetworkImageView;
+
+/**
+ * used by the detail view to display related posts, which can be either local (related posts
+ * from the same site as the source post) or global (related posts from across wp.com)
+ */
+public class ReaderRelatedPostsView extends LinearLayout {
+
+    public interface OnRelatedPostClickListener {
+        void onRelatedPostClick(View v, long siteId, long postId);
+    }
+
+    private OnRelatedPostClickListener mClickListener;
+    private int mFeaturedImageWidth;
+    private ReaderRelatedPostList mRelatedPostList;
+
+    public ReaderRelatedPostsView(Context context) {
+        super(context);
+        initView(context);
+    }
+
+    public ReaderRelatedPostsView(Context context, AttributeSet attrs) {
+        super(context, attrs);
+        initView(context);
+    }
+
+    public ReaderRelatedPostsView(Context context, AttributeSet attrs, int defStyleAttr) {
+        super(context, attrs, defStyleAttr);
+        initView(context);
+    }
+
+    @TargetApi(Build.VERSION_CODES.LOLLIPOP)
+    public ReaderRelatedPostsView(Context context, AttributeSet attrs, int defStyleAttr, int defStyleRes) {
+        super(context, attrs, defStyleAttr, defStyleRes);
+        initView(context);
+    }
+
+    private void initView(Context context) {
+        inflate(context, R.layout.reader_related_posts_view, this);
+        mFeaturedImageWidth = DisplayUtils.dpToPx(context, getResources().getDimensionPixelSize(R.dimen.reader_related_post_image_width));
```

Comment:
I the same vain as the previous comment in this function, perhaps we better use the provided `context` to retrieve the Resources. So, something like `context.getResources()`.  WDYT?